CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Minor: Remove TwitchEmotes.com attribution and the open/copy options when right-clicking a Twitch Emote. (#3136)
```

Comment:
Since including other relevant PRs / issues is welcomed, maybe including #2214 makes sense in this one.
```suggestion
- Minor: Remove TwitchEmotes.com attribution and the open/copy options when right-clicking a Twitch Emote. (#2214, #3136)
```